tests/runner/src/environment/k8.rs

```diff
@@ -1,89 +1,35 @@
-use async_trait::async_trait;
+use crate::{TestOption, load_tls};
+use fluvio_cluster::{ClusterUninstaller, ClusterConfig, ClusterInstaller};
 
-use fluvio_system_util::bin::get_fluvio;
-
-use crate::TestOption;
-use super::EnvironmentDriver;
-use fluvio_command::CommandExt;
-
-pub struct K8EnvironmentDriver {
-    option: TestOption,
+pub async fn remove_k8_cluster() {
+    let uninstaller = ClusterUninstaller::new().build().unwrap();
+    uninstaller.uninstall().await.unwrap();
 }
 
-impl K8EnvironmentDriver {
-    pub fn new(option: TestOption) -> Self {
-        Self { option }
-    }
-}
+pub async fn start_k8_cluster(option: &TestOption) {
+    let mut builder = ClusterConfig::builder(crate::VERSION);
+    builder
+        .spu_replicas(option.spu)
+        .skip_checks(option.skip_checks())
+        .save_profile(true);
 
-#[async_trait]
```

Comment:
Please keep driver trait so we can build unified test runner and make it pluggable